compiler/src/iree/compiler/Codegen/LLVMGPU/test/nvvm_pipeline_test.mlir

```diff
@@ -506,21 +494,7 @@ hal.executable @mma_fused {
 //          MMASYNC:   nvvm.cp.async.wait.group 3
 //  MMASYNC-COUNT-4:   nvvm.ldmatrix{{.*}} : (!llvm.ptr<f32, 3>) -> !llvm.struct<(i32, i32)
 //  MMASYNC-COUNT-8:   nvvm.mma.sync
-//  MMASYNC-COUNT-2:   nvvm.cp.async.shared.global {{.*}}, {{.*}}, 16
-//          MMASYNC:   nvvm.cp.async.commit.group
-//          MMASYNC:   llvm.br
```

Comment:
the commit and br should still be there?

Reply:
Yup!